lambdas/shared/t4_lambda_shared/preview.py

```diff
@@ -2,8 +2,12 @@
 Shared helper functions for generating previews for the preview lambda and the ES indexer.
 """
 from io import BytesIO
+import os
+import json
 import zlib
 
+# number of bytes we take from each document before sending to elastic-search
```

Comment:
Need to update the comment, now that it's used by the preview endpoint. (Possibly rename the variable, too,  if it's not just docs?)

Reply:
By design, elastic and the catalog have orthogonal requirements. I've renamed them accordingly as `CATALOG_LIMIT_*` and `ELASTIC_LIMIT_*`.